src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier color = new ColorSupplier();
+        Random value = new Random();
+        int num = value.nextInt(100);
```

Comment:
replace 100 with constant
```suggestion
        int num = value.nextInt(100);
```